README.md

```diff
@@ -18,6 +18,18 @@
 go get -u github.com/ChainSafeSystems/gossamer
 ```
 
+## Usage 
+
+```
+make gossamer
+gossamer --config config.toml
+```
+
+#### Docker
+```
+make docker
+```
```

Comment:
I think this might be a bit more informative:

#### Docker
To start Gossamer in a docker container, run:
```
make docker
```
##### Running Manually
To build the image, run this command:
```
docker build -t chainsafe/gossamer -f Docerfile.dev
```
Start an instance with:
```
docker run chainsafe/gossamer
```